src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -526,7 +526,8 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_arctanh)
 
 // gamma
 MXNET_OPERATOR_REGISTER_UNARY(gamma)
-.MXNET_DESCRIBE("The gamma function (extension of the factorial function), element-wise")
+.MXNET_DESCRIBE("Returns element-wise gamma function (extension of the factorial function)"
```

Comment:
This isn't grammatical "gamma function value of the input value".

"Returns the gamma function (extension of the factorial function to the reals), computed element-wise on the input array."